tests/tape/interfaces/test_qnode_tf.py

```diff
@@ -23,7 +23,12 @@
 
 
 @pytest.mark.parametrize(
-    "dev_name,diff_method", [["default.qubit", "finite-diff"], ["default.qubit.tf", "backprop"]]
+    "dev_name,diff_method", [

```

Comment:
For my understanding: here we'd still look to add the [jacobian tests](https://github.com/PennyLaneAI/pennylane/blob/c1c8e73af14192fcba03b264f1b5e3850898cc8a/tests/tape/tapes/test_rewind.py#L80) from this branch? But for this PR, we don't need to worry about QNode-level tests due to these paramatrizations?

Reply:
I just wanted a quick way to ensure that this device-level MVP would work on all interfaces, so added the parametrizations here.

But I think these parametrization should stay, they are quite useful.